src/js/media-controller.ts

```diff
@@ -273,6 +276,14 @@ class MediaController extends MediaContainer {
     setBooleanAttr(this, Attributes.NO_DEFAULT_STORE, value);
   }
 
+  get language(): string | undefined {
```

Comment:
```suggestion
  get lang(): string | undefined {
```

Reply:
Not the biggest fan of making function names obscure, I just like things to be more explicit. Anyway, those loose opinions.